src/client-apps/nuxeo-drive/nuxeo-drive-functional-overview/nuxeo-drive-direct-transfer.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Direct Transfer
+description: 'Direct Transfer adds a third core capability to Drive: one-time transfer of content between the Nuxeo Server and the user desktop.'
+review:
+    comment: ''
+    date: '2020-07-20'
+    status: ok
+labels:
+    - nuxeo-drive
+    - direct-transfer
+tree_item_index: 100
+toc: true
+---
+
+The purpose of the Direct Transfer feature is to add a third core capability to Nuxeo Drive: one-time transfer of content between the Nuxeo Server and the user desktop.
+
+Direct Transfer allows users having installed Nuxeo Drive to upload new content to the server, as opposed to using the traditional document creation from Nuxeo Web UI. This will be particularly useful in the case of big files, as timeouts occur with Web UI after a certain while.
```

Comment:
```suggestion
Direct Transfer allows users having installed Nuxeo Drive to upload new content to the server, as opposed to using the traditional document creation from Nuxeo Web UI. This will be particularly useful in the case of big files, as timeouts occur with Web UI after a certain while. This is also different from what is done in the core of Nuxeo Drive: transfers are not tight to [synchronization roots|LINK?] but everywhere on the server (modulo user access rights).
```

Reply:
"as timeouts occur with Web UI after a certain while" can be removed. No need to talk about this. And it has been improved :)